src/coreComponents/fileIO/Table/TableData.cpp

```diff
@@ -49,32 +50,23 @@ std::vector< string > const & TableData::getErrorMsgs() const
   return m_errorsMsg;
 }
 
-void TableData::addErrorMsgs( string const & msg )
+void TableData::addErrorMsg( string const & msg )
 {
-  std::vector< string > splitHeaderParts;
-  std::istringstream ss( msg );
-  string splitErrors;
-
-  while( std::getline( ss, splitErrors, '\n' ))
-  {
-    m_errorsMsg.push_back( splitErrors );
-  }
+  m_errorsMsg.push_back( msg );
```

Comment:
Maybe inline it now? (Or even remove it: it seems to be used once for a `push_back`...)

Reply:
function deleted